webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/Boot.scala

```diff
@@ -709,19 +709,19 @@ class Boot extends Loggable {
     def utilitiesMenu = {
       // if we can't get the workflow property, default to false
       // (don't give rights if you don't know)
-      def workflowEnabled = RudderConfig.configService.rudder_workflow_enabled().either.runNow.getOrElse(false)
-      (Menu(MenuUtils.utilitiesMenu, <i class="fa fa-wrench"></i> ++ <span>Utilities</span>: NodeSeq) /
-      "secure" / "utilities" / "index" >>
-      TestAccess(() => {
-        if (
-          (workflowEnabled && (CurrentUser.checkRights(AuthorizationType.Validator.Read) || CurrentUser
-            .checkRights(AuthorizationType.Deployer.Read))) || CurrentUser
-            .checkRights(AuthorizationType.Administration.Read) || CurrentUser.checkRights(AuthorizationType.Technique.Read)
-        ) {
-          Empty
-        } else {
-          Full(RedirectWithState("/secure/index", redirection))
-        }
+      (Menu(
+        MenuUtils.utilitiesMenu,
+        <i class="fa fa-wrench"></i> ++ <span>Utilities</span>: NodeSeq
+      ) /
+      "secure" / "utilities" / "index"
+      >> LocGroup("utilitiesGroup")
+      >> TestAccess(() => {
+        userIsAllowed(
+          "/secure/index",
+          AuthorizationType.Administration.Read,
```

Comment:
You still need to have workflow enabled checked. the menu should not appaear if workflow enabled is false and we don't have administrator Read. You'll still have the 404 in this case

This pr should only be removal of Technique.Read right